docs/source/rules/examples/REQ-E004/old 2.yaml

```diff
@@ -0,0 +1,21 @@
+swagger: '2.0'
```

Comment:
is this file needed? Is it possible that you've added in the repo files that should not be committed?